keps/sig-apps/4017-pod-index-label/README.md

```diff
@@ -411,7 +411,7 @@ existing label which other things may depend on, for example).
 - Docs are clear about what happens if two pods get the same value (it is set by workload controllers, nothing in the API system will prevent collisions from happening).
 
 #### GA
-Fix any potentially reported bugs.
+- Fix report doc bug: https://github.com/kubernetes/kubernetes/pull/125502/
```

Comment:
I would not block it on it. We probably don't need it at all at this point as suggested there: https://github.com/kubernetes/kubernetes/pull/125502#issuecomment-2367450201.

Reply:
It's a minor fix, I honestly wouldn't even mention it here :wink: but I won't object either.

Reply:
removed this to make it simpler